Web/src/components/games/MentalSums/MentalSumsGame.tsx

```diff
@@ -100,44 +120,59 @@ const MentalSumsGame: React.FC<MentalSumsGameProps> = ({
     setTimeout(() => setAnimateWrong(false), 500);
   };
 
-  const checkAns = (newInput: string) => {
-    if (expectedAns === null) return;
-    if (newInput.length < expectedAns.toString().length) return;
-    if (parseInt(newInput, 10) === expectedAns) {
-      setCorrect();
-      incrementScore();
-    } else {
-      setWrong();
-    }
-    nextProblem();
-    setInput('');
-  };
-
+  useEffect(() => {
+    const checkAns = (newInput: string) => {
```

Comment:
Nit: move this outside the `useEffect` so we don't recreate the arrow function everytime?

Reply:
Ok sorry I derped

Reply:
Actually I moved it in to prevent it from recreating at every render. But I'll change to useCallback ba